ni_measurementlink_service/_internal/stubs/proto/ni/measurementlink/discovery/v1/discovery_service.proto

```diff
@@ -82,6 +82,10 @@ message ServiceDescriptor {
   //   - Key: "ni/service.tags"
   //   - Expected format: serialized JSON string of an array of strings
   //   - Example: "[\"powerup\", \"current\"]"
+  // - Programminglanguage
```

Comment:
This is formatted strangely. Since this is capitalized, I would assume that it is English text and expect `// - Programming Language`. If this was a programmatic identifier, I would not expect it to be capitalized.